### ISSUE TITLE
Beta 5 SIM regognize error

### ISSUE THREAD

**Issue by user858753257** [Android-L-Porting-Team/Android-L-Mako#85]
The Beta 5 doesnt recognize my Vodafone.de Sim card. Has anyone an idea ?

**Comment by HugoNexus4** [comment]
Beta 5 ?

**Comment by user858753257** [comment]
My Nexus didnt see the sim card in the device.

Von: HugoNexus4 [mailto:notifications@github.com] 
Gesendet: Montag, 14. Juli 2014 22:09
An: Android-L-Porting-Team/Android-L-Mako
Cc: F3000
Betreff: Re: [Android-L-Mako] Beta 5 SIM regognize error (#85)

Beta 5 ?

**Comment by jonzey231** [comment]
There is no official Beta 5 release, dude.

**Comment by CTKnight** [comment]
yes !that's what i experienced last 24 hours!I'm on beta4 and my phone keep reminding me of no SIM!

**Comment by user858753257** [comment]
Sorry I mean beta 4

Von: CTKnight
Gesendet: Dienstag, Juli 15, 2014 03:54
An: Android-L-Porting-Team/Android-L-Mako
Cc: F3000
Betreff: Re: [Android-L-Mako] Beta 5 SIM regognize error (#85)

yes !that's what i experienced last 24 hours!I'm on beta4 and my phone keep reminding me of no SIM!

**Comment by CTKnight** [comment]
Didn't experience this bug on beta3pre5 maybe a kernel bug.. Anyway I return to beta3pre5
